controllers/articles.controllers.js

```diff
@@ -34,3 +34,13 @@ exports.patchArticleById = (req, res, next) => {
     })
     .catch((err) => next(err));
 };
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((users) => {
+    let userArr = [];
+    users.forEach((user) => {
+      userArr.push(user);
```

Comment:
Do you need this...? Wouldn't you be able to just sent {users} off in your response?